openwisp_utils/qa.py

```diff
@@ -181,6 +186,39 @@ def check_commit_message():
         sys.exit(1)
 
 
+def check_rst_files():
+    string_io = io.StringIO()
+    settings_overrides = {}
+    settings_overrides.update(readme_rst.SETTINGS)
+    settings_overrides['report_level'] = 2
+    settings_overrides['warning_stream'] = string_io
+    files = [_ for _ in iglob('./**/*.rst', recursive=True)]
+    for file in files:
+        data = read_rst_file(file)
+        try:
+            docutils.core.publish_string(data, settings_overrides=settings_overrides)
+            clean_body = readme_rst.clean(''.join(data))
+            if clean_body is None:
+                raise ValueError("Output Failed")
+        except ValueError:
+            sys.exit(1)
+        except Exception:
+            pass
```

Comment:
@KapilBansal320 can you please add comments here to explicitly mention why `ValueError` is raised. Also. if `ValueError` is encountered the script will exit abruptly and will not tell user what caused the event and the user will think the script is broken. Please make it print a relevant on the terminal.